doc/nrf/ug_thread_tools.rst

```diff
@@ -34,25 +34,236 @@ See `nRF Thread Topology Monitor`_ for documentation.
+
+The OpenThread Border Router is compatible with Nordic Semiconductor devices.
+It implements a number of features, including:
+
+* Bidirectional IP connectivity between Thread and Wi-Fi or Ethernet networks (or both)
+* Network components that allow Thread nodes to connect to IPv4 networks (NAT64, DNS64)
+* Bidirectional DNS-based service discovery using mDNS (on Wi-Fi or Ethernet link, or both) and SRP (on Thread network)
+* External Thread commissioning (for example, using a mobile phone) to authenticate and join a Thread device to a Thread network
+
+You can either install the OpenThread Border Router on a Raspberry Pi or run it using a Docker container, as described in the following sections.
+In both cases, you must first configure a radio co-processor (RCP), which provides the required radio capability to your Linux device.
+
+.. _ug_thread_tools_tbr_rcp:
+
+Configuring a radio co-processor
+================================
+
+The OpenThread Border Router must have physical access to the IEEE 802.15.4 network that is used by the Thread protocol.
+As neither the Linux-based PC nor the Raspberry Pi have such radio capability, you must connect an external nRF device that serves as radio co-processor.
+
+To program the nRF device with the RCP application, complete the following steps:
+
+#. Clone the OpenThread nRF528xx platform repository into the current directory:
+
+   .. code-block:: console
+
+      git clone --recursive https://github.com/openthread/ot-nrf528xx.git
+
+#. Enter the :file:`ot-nrf528xx` directory:
+
+   .. code-block:: console
+
+      cd ot-nrf528xx
+
+#. Install the OpenThread dependencies:
+
+   .. code-block:: console
+
+      ./script/bootstrap
+
+#. Build the RCP example for the hardware platform and the transport of your choice:
+
+   .. tabs::
+
+      .. tab:: nRF52840 Dongle (USB transport)
+
+         .. code-block:: console
+
+            rm -rf build
+            script/build nrf52840 USB_trans -DOT_BOOTLOADER=USB -DOT_THREAD_VERSION=1.2
+
+      .. tab:: nRF52840 Development Kit (UART transport)
+
+         .. code-block:: console
+
+            rm -rf build
+            script/build nrf52840 UART_trans -DOT_THREAD_VERSION=1.2
+
+   ..
+
+   This creates an RCP image at :file:`build/bin/ot-rcp`.
+#. Convert the RCP image to hexadecimal format:
+
+   .. code-block:: console
+
+      arm-none-eabi-objcopy -O ihex build/bin/ot-rcp build/bin/ot-rcp.hex
+
+#. Depending on the hardware platform, complete the following steps:
+
+   .. tabs::
+
+      .. tab:: nRF52840 Dongle (USB transport)
+
+         a. Install nRF Util:
+
+            .. code-block:: console
+
+               python3 -m pip install -U nrfutil
+
+            .. note::
+
+               The nRF Util version distributed officially through PyPI is not supported on Raspberry Pi.
```

Comment:
Wait - are we telling users first how to install nrfutil, and then that this installation doesn't work and they need to use a different way?
Or am I missing something?

Reply:
AFAIK nrfutil doesn't work on Raspbian, because of the one dependency wrong version called pc_ble_driver. I think that  by following this instruction after calling `python3 -m pip install -U nrfutil` we will have all necessary dependencies installed, but one of them is wrong, so we need to install it manually and switch version later on. @LuDuda, correct?

Reply:
That is correct. It's quite unlikely that someone will build RCP image on RaspberryPi rather than on Linux-based PC, and so I decided just to put a note below.

Reply:
Hmm, a bit skeptical about that ... I read this in the way that you either use a Raspberry Pi for everything or a Linux PC for everything. But maybe that is clearer when you actually do it.

Reply:
One option to mitigate this: make it clearer in the note that it concerns Raspbian:
```suggestion
               If you are using Raspberry Pi, the nRF Util version distributed officially through PyPI is not supported on Raspbian OS.
To install a compatible version, execute the following commands:
```

Reply:
Agreed F2F change to:

```
If you are using Raspberry Pi, the nRF Util version distributed officially through PyPI is not supported on Raspbian OS.

To install a compatible version, execute the following commands:
```

Reply:
Actually:
```
If you are using a Raspberry Pi, the nRF Util version distributed officially through PyPI is not supported.
To install a compatible version on Raspbian OS, execute the following commands:
```